src/Components/OrganizationData/OrganizationDataLayout/OrganizationDataLayout.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import  { Box, Container, Button} from '@material-ui/core';
+
+const OrganizationDataLayout = ({organizationData}) => {
```

Comment:
The organization information should come from the endpoint [get organization](http://ongapi.alkemy.org/api/docs#/Organization/get_organization) instead of a parameter. For that, you should have an OrganizationService that contain getOrganization and call that endpoint inside of it.
the user story say: " se mostrarán los datos actuales de la organización " so we should always get the data from the endpoint to be sure that we are showing the right data.

Reply:
I create a service and use call it on the component. Ty